apps/web/pages/new/digitalocean.tsx

```diff
@@ -0,0 +1,219 @@
+import { Bucket, ListBucketsCommandOutput } from "@aws-sdk/client-s3";
+import {
+	Box,
+	Button,
+	Container,
+	Divider,
+	Flex,
+	Heading,
+	IconButton,
+	Input,
+	Select,
+	Text,
+} from "@chakra-ui/react";
+import VideoModal from "@components/ui/VideoModal";
+import useUser from "@hooks/useUser";
+import axios from "axios";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import toast from "react-hot-toast";
+import { ArrowNarrowLeft } from "tabler-icons-react";
+import "video-react/dist/video-react.css";
+import validator from "validator";
+
+const NewS3 = () => {
```

Comment:
This class follows the structure of other classes in this directory. Can we leverage a parent class (aka a base class) with inheritable methods so we do not need to repeat the same patterns?

Just a thought to chew on.